src/cdk-experimental/listbox/listbox.ts

```diff
@@ -171,15 +172,17 @@ export class CdkOption implements ListKeyManagerOption, Highlightable {
     host: {
       'role': 'listbox',
       '(keydown)': '_keydown($event)',
-      '[attr.aria-disabled]': '_disabled',
-      '[attr.aria-multiselectable]': '_multiple',
+      '[attr.tabindex]': 'tabIndex',
+      '[attr.aria-disabled]': 'disabled',
+      '[attr.aria-multiselectable]': 'multiple',
       '[attr.aria-activedescendant]': '_getAriaActiveDescendant()'
     }
 })
 export class CdkListbox implements AfterContentInit, OnDestroy, OnInit {
 
   _listKeyManager: ActiveDescendantKeyManager<CdkOption>;
   _selectionModel: SelectionModel<CdkOption>;
+  tabIndex = 0;
```

Comment:
Should this be prefixed with an underscore? (similarly to how we do it within the selection list)